CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 ## Unreleased
 
+## 1.23.0 / 2025-07-15
+
+* [CHANGE] Minimum required Go version is now 1.23, only the two latest Go versions are supported from now on. #1812
+* [FEATURE] Add WrapCollectorWith and WrapCollectorWithPrefix #1766
+* [FEATURE] Add exemplars for native histograms #1686
+* [ENHANCEMENT] exp/api: Bubble up status code from writeResponse #1823
+* [ENHANCEMENT] Update runtime metrics for Go v1.23 and v1.24 #1833
+* [BUGFIX] fix: client prompt return on context cancellation #1729
+* [BUGIFX] Fix errNotImplemented reference #1835
```

Comment:
Let's double check if this is a bug exists in the previous released version. If it is not no need to mention it in the release notes.